samples/tfm_integration/psa_level_1/sample.yaml

```diff
@@ -5,7 +5,8 @@ sample:
 tests:
     sample.tfm_ipc:
```

Comment:
Not related to current change, but might deserve a fix

Reply:
Fixing that too, thanks.